cni/pkg/nodeagent/informers.go

```diff
@@ -103,6 +103,25 @@ func (s *InformerHandlers) GetPodIfAmbient(podName, podNamespace string) (*corev
 	return nil, nil
 }
 
+// GetPodIfAmbientActive looks up a pod. It returns:
+// * An error if the pod cannot be found
+// * nil if the pod is found, but is not enrolled in ambient (e.g. has been annotated)
+// * the pod, if it is found and is enrolled in ambient (e.g. has been annotated)
+func (s *InformerHandlers) GetPodIfAmbientActive(podName, podNamespace string) (bool, error) {
+	ns := s.namespaces.Get(podNamespace, "")
+	if ns == nil {
```

Comment:
nit: same with GetPodIfAmbientEnabled, is checking ns necessary? 
When a ns is deleting, a pod within it will be deleted too becasue of cascading delete

Reply:
- `GetPodIfAmbientActive` is actually unused now (which the linter didn't flag/I missed) so removing that, ty.

- `GetPodIfAmbientEnabled` - `is checking ns necessary?  When a ns is deleting, a pod within it will be deleted too becasue of cascading delete` ->

Yes - that is why we check cache for the NS first, and return early if we can't get it (before bothering to check for the pod).